### ISSUE TITLE
It can automatic monitor incoming calls?

### ISSUE THREAD

**Issue by DearZack** [aykuttasil/CallRecorder#1]

**Comment by luongvo** [comment]
Yes, of course!

**Comment by GlennPallad** [comment]
Yes, I've tested it on MIUI, works pretty good, very succinct app.

**Comment by levirgon** [comment]
is the sample app working? where do i find the saved file? do i have to click the record button during a call?

**Comment by shashank-shinde** [comment]
Use `setAudioSource(MediaRecorder.AudioSource.VOICE_CALL)` instead `setAudioSource(MediaRecorder.AudioSource.VOICE_COMMUNICATION)`
